quickstart.md

```diff
@@ -0,0 +1,131 @@
+[ &larr; back](README.md)
+<br/>
+# Quickstart
+
+This is a quickstart guide on how to create a new Zendro project with default parameters. It uses pre-defined datamodels, database and environment variables.
+
+If you want to know more about Zendro or a detailed explanation on how to set up Zendro from scratch, check [this](setup_root.md).
+
+ <br/>
+
+## Project Requirements:
+ * [NodeJS](https://nodejs.org/en/)
+ * [docker](https://docs.docker.com/get-docker/)
+ * [docker-compose](https://docs.docker.com/compose/install/#install-compose)
+ <br/><br/>
+
+* * *
+### Step 1: Installation
+
+Execute this commands to install Zendro:
+
+```
+$ git clone https://github.com/Zendro-dev/zendro.git
+$ cd zendro
+$ npm install
+$ npm link
+```
+
+### Step 2: Setup a new Zendro project
+
+The easiest way to set up Zendro is using the [Zendro CLI tool](https://github.com/Zendro-dev/zendro) with minimal steps and configuration. Execute:
+
+```
+$ zendro set-up -d <name>
+```
+
+By default, three data models with associations will be used for this instance:
+* city
+* country
+* river
+
+Also a default SQLite database will be used. You can find the database on *graphql-server* folder.
+
+### Step 3: Edit environment variables
+
+Edit *NEXTAUTH_SECRET* to your expected secret word in the following files:
+* **SPA in development mode:** ./single-page-app/.env.development
+* **SPA in production mode:** ./single-page-app/.env.production
+* **GraphiQL in development mode:** ./graphql-server/.env.development
+* **GraphiQL in production mode:** ./graphql-server/.env.production
+
+If you want to know more about the enviroment variables, you can check [this](env_vars.md).
+
+### Step 4: Start up your Zendro instance
+
+Execute the next command to start Zendro in production mode. 
+
+```
+$ zendro dockerize -u -p
+```
+
+This command will create docker containers for each Zendro component:
+* [Keycloak](https://github.com/Zendro-dev/Zendro-dev.github.io/blob/documentation-vb/oauth.md): manage users and roles
+* [Single Page App (SPA)](https://github.com/Zendro-dev/single-page-app): graphical interface to send CRUD requests to a Zendro GraphQL endpoint
+* [API](https://github.com/Zendro-dev/graphql-server): CRUD API that can be accessed through a GraphQL query language
+* [API with authenthication](https://github.com/Zendro-dev/graphiql-auth): An implementation of the GraphQL IDE with Zendro login
```

Comment:
API with authenthication => API with filter functionality
The main purpose of graphiql is to filter the result by jq or JSONPath